### DISCUSSION TITLE
Новый стиль у полей.

### DISCUSSION THREAD

**Discussion by MadZee-M** [moonshine-software/moonshine#1128] [Ideas]
Доброго времени суток!

Предлагаю добавить grid стиль для полей.

![image](https://github.com/user-attachments/assets/e919226f-689c-4663-804a-60adcf0a4d0a)

Идея следующая:

1. Добавить новый метод styleGrid() или grid()
2. Аргументы: ($colSpan = 9, $adaptiveColSpan = 12) (по умолчанию).
3. colSpan для label - $adaptiveColSpan - $colSpan
4. Ну и верстка.

И добавив один метод, будет красивый вид.

Также сделать такой метод для Группы полей. И будет применяться ко всем полям (чтобы не клонировать на 100500 полей один и тот же метод)

**Comment by lee-to** [comment]
А почему вам просто не сделать свое поле типа и просто применять его аля FlexField::make(Text::make('Text')) и итог будет как вы ожидаете

Муншайн дает все возможности чтобы такое реализовать с минимальными навыками разработчика

Я не буду закрывать эту дискуссию, возможно кто то из комьюнити захочет сделать подобный пр, но основная команда Муншайн врядли будет делать в ближайшее время такую фичу

**Comment by lee-to** [comment]
https://github.com/moonshine-software/moonshine/pull/1138
